src/actions/update-solution-version/index.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import * as core from '@actions/core';
+import { DefaultRunnerFactory, getWorkingDirectory, RunnerFactory } from '../../lib';
+import path = require('path');
+
+(async () => {
+    if (process.env.GITHUB_ACTIONS) {
+        await main(DefaultRunnerFactory);
+    }
+})();
+
+export async function main(factory: RunnerFactory): Promise<void> {
+    try {
+        core.startGroup('update-solution-version:');
+        const solutionPatchVersion = core.getInput('solution-patch-version', { required: false });
+        const solutionVersionUpdateStrategy = core.getInput('solution-version-update-strategy', { required: false });
+
+        if (!!solutionPatchVersion === !!solutionVersionUpdateStrategy) {
+            const error = new Error('Either provide solution-patch-version or solution-version-update-strategy');
```

Comment:
improve the error message, yes, there isn't much room for broader feature explanation in an error msg...
e.g.
Cannot specify an explicit version (via solution-patch-version) and a versioning strategy (via solution-version-update-strategy) like gittags or filetracker